src/handlers/indexers/utils.rs

```diff
@@ -9,3 +9,70 @@ pub fn get_s3_script_key(id: Uuid) -> String {
 pub fn get_script_tmp_directory(id: Uuid) -> String {
     format!("{}/{}/{}.js", env!("CARGO_MANIFEST_DIR"), "tmp", id)
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Do these tests really help?

1. We are using the same code inside (which is used inside the function) and in the test case so it's bound to match.
2. If I change the directory later on (which I did in the last PR), I don't really want to change all these test cases as well

I think the directory building doesn't check a very important logic to write test cases for them.